README.md

```diff
@@ -101,6 +101,14 @@ In IntelliJ/Cursive, you can [set up REPL commands](https://cursive-ide.com/user
 * make sure the command is executed in the `nextjournal.clerk` namespace;
 * lastly assign a shortcut of your choice via `Settings→Keymap`
 
+#### Neovim + Conjure
+
+Using the `expand(%)` vim expression to get the current filepath, [neovim](https://neovim.io/) + [conjure](https://github.com/Olical/conjure/) users can map `:execute ":ConjureEval (clerk/show! \"" . expand("%") . "\")"` to a keybinding:
+
+```
+nmap <silent> <localleader>cs :execute ":ConjureEval (clerk/show! \"" . expand("%") . "\")"<CR>
```

Comment:
I guess we want to namespace this to `nextjournal.clerk` so it also works without a clerk require / alias?